client/src/components/navbar/index.js

```diff
@@ -15,9 +14,7 @@ export default class navbar extends React.Component {
           <StyledButton name="events" url="/events/" />
           <StyledButton name="info" url="/info/" />
         </div>
-        <Route path="/info/" component={ParentInfo} />
-        <Route path="/events/" component={Events} />
-        <Route path="/profile/" component={ParentProfile} />
+
       </Router>
```

Comment:
if you deleted the Routes Why you kept the Router ?

Reply:
They'll be readded and <route> will be replaced with Link because navigations needs the router, for now I don't need navigation so i removed the routes and kept the router for the <Link> tags.

Reply:
Yes.